ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/services/HDFS/metainfo.xml

```diff
@@ -223,6 +223,52 @@
           </configFiles>
         </component>
 
+        <component>
+          <name>ROUTER</name>
+          <displayName>Router</displayName>
+          <category>SLAVE</category>
+          <cardinality>0+</cardinality>
+          <versionAdvertised>true</versionAdvertised>
+          <reassignAllowed>true</reassignAllowed>
+          <timelineAppid>router</timelineAppid>
+          <commandScript>
+            <script>scripts/router.py</script>
+            <scriptType>PYTHON</scriptType>
+            <timeout>1800</timeout>
+          </commandScript>
+          <logs>
+            <log>
+              <logId>hdfs_router</logId>
+              <primary>true</primary>
+            </log>
+            <log>
+              <logId>hdfs_audit</logId>
+            </log>
+          </logs>
+          <dependencies>
+            <dependency>
+              <name>HDFS/HDFS_CLIENT</name>
+              <scope>host</scope>
+              <auto-deploy>
+                <enabled>true</enabled>
+              </auto-deploy>
+            </dependency>
+          </dependencies>
+          <configuration-dependencies>
+            <config-type>hdfs-rbf-site</config-type>
```

Comment:
making hdfs-rbf-site as a config dependency for router only so that any changes in the file do not affect the HDFS cluster.